docker-vms/vumi_metrics/config/supervisor/vumi_metrics.conf

```diff
@@ -0,0 +1,62 @@
+[supervisord]
+nodaemon=true
+
+[program:graphite_metrics_collector]
+command=twistd -n --pidfile=graphite-metrics-collector-%(process_num).pid \
+    vumi_worker \
+    --worker-class=vumi.blinkenlights.GraphiteMetricsCollector \
+    --hostname=rabbitmq --vhost=/ --username=guest --password=guest
+directory=/app/
+environment=
+autostart=true
+autorestart=true
+stderr_logfile_maxbytes=10MB
+stdout_logfile_maxbytes=10MB
+stdout_logfile_backups=10
+stderr_logfile_backups=10
+stderr_logfile=/app/graphite_metrics_collector.err
+stdout_logfile=/app/graphite_metrics_collector.log
+
+[program:metric_time_buckets]
+command=twistd -n --pidfile=metric-time-buckets-%(process_num)s.pid \
+    vumi_worker \
+    --worker-class=vumi.blinkenlights.MetricTimeBucket \
+    --hostname=rabbitmq --vhost=/ --username=guest --password=guest \
+    --config=metric_time_buckets.yaml \
+    --set-option=buckets:3
+numprocs=2,
```

Comment:
We can have any number. They have to process all metrics one by one and fire them off to their respective time buckets, so I thought having more than one was a good idea. Not wildly important either way, I think.